funcx_endpoint/funcx_endpoint/endpoint/interchange.py

```diff
@@ -220,20 +209,34 @@ def load_config(self):
             self.executors[executor.label] = executor
             if executor.run_dir is None:
                 executor.run_dir = self.logdir
+
+    def start_executors(self):
+        logger.info("Starting Executors")
+        for executor in self.config.executors:
```

Comment:
I moved this into a new method so that executor does not start in the init method of the class (this start method gets called in the interchange `start` instead)

Reply:
This is reasonable.